app/src/main/java/com/duckduckgo/app/global/view/DaxDialog.kt

```diff
@@ -161,10 +161,14 @@ class TypewriterDaxDialog : DialogFragment(), DaxDialog {
             dismiss()
         }
 
-        if (dismissible) {
-            dialogContainer.setOnClickListener {
+        dialogContainer.setOnClickListener {
+            if (dismissible) {
```

Comment:
1. Do we have any dax dialog that is dismissible right now? It seems now all of them are not dismissible, maybe we should change the default configuration for those dialogs and have `private var dismissible: Boolean = false` instead of `true`.

2. should we dismiss dialog if user presses back button? currently we dismiss it.

Reply:
1. Good shout! I've made it `false` by default and removed the noise in the constructors.

2. Good question. We didn't discuss that but I've added it to https://app.asana.com/0/715106103902962/1199649920028917 which is a task for further optimizations. I think we can go ahead as it is.